Splash

robert5974 opened the pull request:
## Description: Adding a SplashScreen to the dash application.


**Related issue (if applicable):** fixes #<issue number goes here>


## Checklist:
  - [ ] The code change is tested and works locally.

robert5974 committed:
Update Readme.md

Updated Readme with OpenDash specific directions and included a future section for when the Wiki is created.

robert5974 committed:
Update README.md

robert5974 committed:
Add files via upload

robert5974 committed:
Add files via upload

robert5974 committed:
Update README.md

robert5974 committed:
Update README.md

robert5974 committed:
Update README.md

robert5974 committed:
Update README.md

robert5974 committed:
Update README.md

robert5974 committed:
Update ReadMe

Updated Openauto widget image

robert5974 committed:
Update ReadMe.md

Updated Openauto Widget Music image

robert5974 committed:
Update ReadMe.md

Added install script instructions

robert5974 committed:
Create dash.cpp

robert5974 committed:
Create splash.png

robert5974 committed:
Create splash_highres1.png

robert5974 committed:
Create splash_highres2.png

robert5974 committed:
Create splash_highres3.png

robert5974 committed:
Update splash.png

robert5974 committed:
Delete splash_highres1.png

robert5974 committed:
Delete splash_highres2.png

robert5974 committed:
Delete splash_highres3.png

robert5974 committed:
Update resources.qrc

robert5974 committed:
Update dash.cpp

robert5974 committed:
Update splash.png

robert5974 committed:
Update ReadMe.md

Updated Features list with updated information

robert5974 committed:
Update ReadMe.md

Updated intro to specify this is the Dash portion of the project.

robert5974 committed:
Remove manual build instructions and added line

Moving manual build instructions to the wiki.  Added line for brief explanation of install script.

robert5974 committed:
Merge remote-tracking branch 'upstream/develop' into develop

robert5974 committed:
Merge branch 'develop' into splash

robert5974 committed:
Create camera_tab_dark.png

robert5974 committed:
Delete data.png

robert5974 committed:
Create data_tab_dark.png

robert5974 committed:
Delete home.png

robert5974 committed:
Delete IA_applaunchtab_light_theme.PNG

robert5974 committed:
Delete IA_camstrmtab_light_theme.PNG

robert5974 committed:
Delete IA_datatab_light_theme.PNG

robert5974 committed:
Delete IA_mediaplayer_light_theme.PNG

robert5974 committed:
Delete IA_openauto_tab_light_theme_maps.PNG

robert5974 committed:
Delete IA_openauto_tab_light_themeYTmusic.PNG

robert5974 committed:
Delete IA_openauto_tab_USB.PNG

robert5974 committed:
Delete IA_openauto_tab_USB_connectedmaps.PNG

robert5974 committed:
Delete IA_openauto_tab_USB_connectedmusic.PNG

robert5974 committed:
Delete IA_settings_general.PNG

robert5974 committed:
Delete IA_settings_layoutpage.PNG

robert5974 committed:
Delete IA_settings_light_theme.PNG

robert5974 committed:
Delete IA_settings_openautopage.PNG

robert5974 committed:
Delete media.png

robert5974 committed:
Create mediaplayer_tab_dark.png

robert5974 committed:
Create openauto_tab_dark.png

robert5974 committed:
Create opendash-ui.gif

robert5974 committed:
Delete settings.png

robert5974 committed:
Create settings_tab_main_dark.png

robert5974 committed:
Update README.md

robert5974 committed:
Update README.md

robert5974 committed:
Update README.md

rsjudka commented on src/dash.cpp:
```diff
@@ -6,6 +6,32 @@
 
 int main(int argc, char *argv[])
 {
+     QApplication dash(argc, argv);
```
looks like something got screwed up with the merge... might want to double check the diff here, looks like your version of the file was "stacked" ontop of the original

robert5974 replied:
Corrected the dash.cpp.  Not sure why the merge combined the files but good catch.

rsjudka commented on assets/resources.qrc:
```diff
@@ -1,6 +1,7 @@
 <RCC>
     <qresource prefix="/">
         <file>aa_video.qml</file>
+	    	<file>splash.png</file>
```
change to spaces instead of tabs (to keep the spacing consistent)

robert5974 replied:
corrected tabs to spaces per comment

rsjudka commented on README.md:
```diff
@@ -1,143 +1,41 @@
-
-`./dash`
-
-Video walk through: 
```
can you leave the link to the video, but maybe just give a disclaimer that the process has changed slightly

rsjudka replied:
ia renamed to dash, older ui, etc

robert5974 replied:
Updated with Video walkthrough...per comments

robert5974 committed:
Added back Video Walkthrough

Accidently removed the video walkthrough so it was added back per PR comment

robert5974 committed:
Updated Readme

The explanation for changes to Video walkthrough was expanded

robert5974 committed:
removed tabs, replaced with spaces

robert5974 committed:
Removed old lines

Merging apparently combined the former and new dash.cpp files into one.  The old code was removed.

robert5974 reviewed (commented):
Corrected with comments.
